src/Components/ComponentRenderer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Components;
+
+use Tempest\Console\Console;
+use Tempest\Console\ConsoleComponent;
+
+interface ComponentRenderer
```

Comment:
Added this interface so that I can easily mock it during testing and for commands that run in the background. Both cases cannot accept user input (for now)